cli/pkg/utils/utils.go

```diff
@@ -93,25 +96,27 @@ func GetTobsChartMetadata(chart string) (*ChartMetadata, error) {
 }
 
 func GetDeployedChartMetadata(releaseName, namespace string) (*DeployedChartMetadata, error) {
-	chartDetails := &DeployedChartMetadata{}
 	out := exec.Command("helm", "list", "--namespace", namespace, "-o", "json")
 	res, err := out.CombinedOutput()
 	if err != nil {
-		return chartDetails, fmt.Errorf("failed to list helm releases %w", err)
+		return nil, fmt.Errorf("failed to list helm releases %w", err)
 	}
-
 	charts := &[]DeployedChartMetadata{}
 	err = json.Unmarshal(res, charts)
 	if err != nil {
-		return chartDetails, fmt.Errorf("failed to unmarshal deployed helm chart metadata %w", err)
+		return nil, fmt.Errorf("failed to unmarshal deployed helm chart metadata %w", err)
 	}
 	for _, c := range *charts {
 		if c.Name == releaseName {
 			return &c, nil
 		}
 	}
 
-	return chartDetails, nil
+	return nil, ErrorTobsDeploymentNotFound()
```

Comment:
Unnecessary to make a function that returns only one error, can be a package var.

Reply:
If I make this a variable, I can't inject the values dynamically as global variables are built in the compile time. 
```
func ErrorTobsDeploymentNotFound() error {
	return fmt.Errorf("unable to find the tobs deployment with name %s in namespace %s", root.HelmReleaseName, root.Namespace)
}
```